src/io/export_pov.cc

```diff
@@ -40,6 +40,9 @@
 void export_pov(const std::shared_ptr<const Geometry>& geom, std::ostream& output, const ExportInfo& exportInfo)
 {
   std::shared_ptr<const PolySet> ps = PolySetUtils::getGeometryAsPolySet(geom);
+  if (!ps->isTriangular()) {
+    ps = PolySetUtils::tessellate_faces(*ps);
+  }
```

Comment:
I believe we discussed this during the initial POV export work: POV export is able to handle non-triangular polygons, and there was some argument why that was intentional. This line only make all polygons into triangles. It's a fine workaround, but it really indicates that there is a bug downstream from here which doesn't handle such polygons.